### PR TITLE
chore: add placeholder project status sidebar

### PR DESCRIPTION
**Pull request by thomasheartman**
This PR adds an empty placeholder sidebar for the project status content.

It also adds a button to open the sidebar.

Additionally, because the button to open the sidebar takes the place of the existing "import" button, the import button has also been moved down to the filter row.

Of course, these changes are all behind the flag, so if nothing should change if the flag is not enabled.

image
image

### CODE REVIEW

File: frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx [hunk 1]
```diff
@@ -66,6 +71,19 @@ const Container = styled('div')(({ theme }) => ({
...
+    flexFlow: 'row wrap',
+    gap: theme.spacing(2),
+    justifyContent: 'space-between',
+}));
+
+const ButtonGroup = styled('div')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(1),
+    paddingInline: theme.spacing(1.5),
+}));
```
Comment (thomasheartman) [inline frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx:85]: This group is there to accomodate the export and column selector buttons which will eventually also move there (according to the sketches)

File: frontend/src/component/project/Project/ProjectStatusModal.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}));
+
+type Props = {
+    open: boolean;
+    close: () => void;
+};
+
+export const ProjectStatusModal = ({ open, close }: Props) => {
+    return (
+        <SidebarModal open={open} onClose={close} label='Import toggles'>
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectStatusModal.tsx:17]: Not sure about the label

Comment (sjaanus) [reply frontend/src/component/project/Project/ProjectStatusModal.tsx:17]: While you are at it, lets rename the original to flags.

**Review by sjaanus — APPROVED** [review]

File: frontend/src/component/project/Project/ProjectStatusModal.tsx (hunk 2 above)
Comment (thomasheartman) [reply frontend/src/component/project/Project/ProjectStatusModal.tsx:17]: Oh, right. That's a copy-paste error. yeah, I'll fix

**Pull request merged**